addons/mail/models/mail_message.py

```diff
@@ -905,53 +923,55 @@ def _message_fetch(self, domain, max_id=None, min_id=None, limit=30):
         return self.search(domain, limit=limit)
 
     def message_format(self, format_reply=True):
-        """ Get the message values in the format for web client. Since message values can be broadcasted,
-            computed fields MUST NOT BE READ and broadcasted.
-            :returns list(dict).
-             Example :
+        """ Get the message values in the format for web client. Since message
+        values can be broadcasted, computed fields MUST NOT BE READ and
+        broadcasted.
+
+        :returns list(dict). Example :
```

Comment:
(minor: I'm not sure if there is added value to keep maintaining this example, but if you like it then it's fine by me).